src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,9 +1,46 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static core.basesyntax.FileCreator.createFile;
```

Comment:
Avoid using static imports for methods; it can make the code less readable because it's not clear where the method comes from. It's better to call the method on the class where it's defined.